package.json

```diff
@@ -511,6 +511,18 @@
         "icon": "$(refresh)",
         "category": "GitHub Pull Requests"
       },
+      {
+        "command": "pr.setFileListLayoutAsTree",
```

Comment:
I think these should be hidden in the command palette, you can do that by adding them in the "commandPalette" section below with "when": "false"